.test-infra/jenkins/job_PostCommit_Python_ValidatesRunner_Dataflow.groovy

```diff
@@ -32,8 +32,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Py_VR_Dataflow', 'Run Python
   steps {
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
-      tasks(':beam-sdks-python:validatesRunnerBatchTests')
-      tasks(':beam-sdks-python:validatesRunnerStreamingTests')
+      tasks(':python3ValidatesRunner')
```

Comment:
Looks like this removes Py2 validates-runner suites? We should keep them.

Reply:
I readded them  (I only removed them to test Jenkins behaviour, turns out `Run Seed Job` is what I needed).